.circleci/config.yml

```diff
@@ -630,6 +630,16 @@ jobs:
             echo "export RGP_USER_PASSWORD=$rgpUserPassword" >> playwright-env/envvars
             echo "export RGP_USER_EMAIL=$rgpUser" >> playwright-env/envvars
 
+            # Read Auth0 RGP client credentials
+            export rgpDomain=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.auth0 | .[] | select(.app==\"rgp\" and .env==\"<< parameters.env >>\") | .domain")
+            export rgpAudience=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.auth0 | .[] | select(.app==\"rgp\" and .env==\"<< parameters.env >>\") | .audience")
+            export rgpClientId=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.auth0 | .[] | select(.app==\"rgp\" and .env==\"<< parameters.env >>\") | .clientId")
+            export rgpClientSecret=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.auth0 | .[] | select(.app==\"rgp\" and .env==\"<< parameters.env >>\") | .clientSecret")
+            echo "export RGP_AUTH0_DOMAIN=rgpDomain" >> playwright-env/envvars
+            echo "export RGP_AUTH0_AUDIENCE=rgpAudience" >> playwright-env/envvars
+            echo "export RGP_AUTH0_CLIENT_ID=rgpClientId" >> playwright-env/envvars
+            echo "export RGP_AUTH0_CLIENT_SECRET=rgpClientSecret" >> playwright-env/envvars
+
```

Comment:
I would recommend instead of extracting this info from `vault` directly to get it from pepperConfig.js file.
Will be less lines of code and we are trying to move to GCP Secret Manager .
Here is one place where we call build-study.sh which generates the pepperConfig.js
https://github.com/broadinstitute/ddp-angular/blob/b2146e488df0f570ffff0eac99b78c227532fcfa/.circleci/config.yml#L176

Reply:
I think that's a different matter which could be addressed in a future PR. I found this [ticket](https://broadworkbench.atlassian.net/browse/PEPPER-352) which could include Playwright test secrets management.

Reply:
I am just pointing out that the info you seek to extract from Vault is in that pepperConfig.js generated by that script. This is how the Angular applications get these exact same values from pepperConfig.js . When we move to Secret Manager we will continue to use pepperConfig.js in Angular apps so it would be stable way to get them

Reply:
Got it. I hope I can update all Playwright secrets to use GCP secret manager in one shot in a small and scoped PR.

Reply:
To confirm my understanding, this change is not required for approval, right?